src/components/Autocomplete.vue

```diff
@@ -197,7 +203,11 @@ export default {
         this.$emit('submit', filter);
         this.q = '';
       }
-    }
+    },
+    filterCount() {
+      return typeof this.filters === 'undefined' ?
```

Comment:
Pro tip: a check like `this.filters == null` catches both cases when `this.filters` is `null` and `undefined`

Reply:
tnks, noted.